### PR TITLE
Vault hotfix 2

### PR DESCRIPTION
**Pull request by zuliouse**
## Description
Deletes the vines that SIPHON OXYGEN FOR SOME REASON. Also some misc changes like fixing the ORM orientation and moving the excavator PA to a more secure location.

## Motivation and Context
No oxygen = bad.

## Changelog (necessary)
:cl:
add: Added a grinder in hydroponics.
del: Removed the vines that suffocate you.
tweak: Moved the excavator PA to the bot control room.
fix: Fixed the ORM not inputting ore from the east tile in the hallway.
/:cl:

### CODE REVIEW

File: code/modules/food_and_drinks/recipes/drinks_recipes.dm [hunk 1]
```diff
@@ -722,10 +722,10 @@
...
 
-/datum/chemical_reaction/fernetnuka
-	name = "Fernet Nuka"
-	id = "fernetnuka"
-	results = list("fernetnuka" = 2)
+/datum/chemical_reaction/fernetcola
+	name = "Fernet Cola"
+	id = "fernetcola"
+	results = list("fernetcola" = 2)
 	required_reagents = list("fernet" = 1, "nuka_cola" = 1)
```
Comment (Laus4Deus) [inline code/modules/food_and_drinks/recipes/drinks_recipes.dm:729]: You changed the `chemical_reaction` result. Did you made a reagent named `fernetcola`? if there isn't one named like that, the liquids will disapear when the reaciton takes plce

Comment (Laus4Deus) [reply code/modules/food_and_drinks/recipes/drinks_recipes.dm:729]: God my grammar. I am studying for an exam damn it!
Fix this and it will be approved! 🔪

Comment (zuliouse) [reply code/modules/food_and_drinks/recipes/drinks_recipes.dm:729]: Yup, I renamed fernetnuka to fernetcola in my drinks PR as part of me restoring fernet cola (after Scar gutted it for no reason), but forgot to rename the `chemical_reaction` so this is me tying up loose ends.

Comment (Laus4Deus) [reply code/modules/food_and_drinks/recipes/drinks_recipes.dm:729]: alright, if then it should be okay.
minus the merge conflict that is

**Pull request closed without merging**